### PR TITLE
CentOS support

### PR DESCRIPTION
**Pull request by mpreisler**
This pull request adds ssg-centos variants next to ssg-rhel XCCDFs and datastreams. These new files have CentOS CPEs and work out of the box on CentOS machines. They contain an informative notice about the change. You can see the notice on the following screenshot of HTML guide generated from the CentOS datastream:

centos_notice

How the files are generated:
1) the build process creates XCCDF, OVAL and datastream files
2) the enable-centos.py script is run on XCCDF and datastream to produce their CentOS variants
3) these new files are copied to dist as part of `make dist`

I would like to thank Russell Doty, Jan Lieskovsky, Simon Lukasik and Shawn Wells for their feedback and contributions to this feature.

Feedback welcome!

### CODE REVIEW

**Comment by isimluk** [comment]
I like how this is done (programming wise) and I think that the language is professional and describes the differences well. However mainly, I welcome ssg to start official distribution of centos content together with rhel content, as a lot of users have been requesting this and they have already been patching their rhel guidance.

But I would wait for Shawn's ack before merging.

**Comment by jan-cerny** [comment]
I agree with @isimluk . I like that we invite CentOS people to participation on SSG in this disclaimer :-) The python script looks great, but I haven't gone through it deeply.

File: shared/utils/enable-centos.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    "benchmark developed for <i>Red Hat Enterprise Linux</i>. It has been " \
+    "modified through an automated process to remove specific dependencies " \
+    "on <i>Red Hat Enterprise Linux</i> and to function with <i>CentOS</i> " \
+    "&#8212; <i>Community Enterprise Operating System</i>. The result is a " \
+    "generally useful <i>SCAP Security Guide</i> benchmark with the following " \
+    "caveats:</p>\n" \
+    "<ul>\n" \
+    "<li><i>CentOS</i> is not an exact copy of " \
+    "<i>Red Hat Enterprise Linux</i>. There may be configuration differences " \
+    "that produce both false positives and false negatives. If this occurs " \
```
Comment (shawndwells) [inline shared/utils/enable-centos.py]: very small readability edit:

````that produce both false positives and false negatives```` --> ````that produce false positives and/or false negatives````

File: shared/utils/enable-centos.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    "please file a bug report.</li>\n" \
+    "\n" \
+    "<li><i>CentOS</i> has its own build system, compiler options, patchsets, " \
+    "and is a community-led operating system, <i>CentOS</i> does not inherit " \
```
Comment (shawndwells) [inline shared/utils/enable-centos.py]: Within many regulated environments, policies exist to favor COTS software. It'd be good to gently remind users that CentOS is not commercial software nor commercially supported. How about:

````.... and is a community supported, non-commercial operating system. <i>CentOS</i> does not....````

Note the new period too.

File: shared/utils/enable-centos.py (hunk 1 above)
Comment (mpreisler) [inline shared/utils/enable-centos.py]: done

File: shared/utils/enable-centos.py (hunk 2 above)
Comment (mpreisler) [inline shared/utils/enable-centos.py]: done

**Comment by shawndwells** [comment]
Merged and ran ````make```` on rhel6 and 7 - no issues. language embedded within CentOS XCCDF, shows in reports. 

Bump to @gregelin for awareness, a vocal requestor of this.
Perhaps @onyxpoint will update his slides to give apologies *and* thanks ;)

**Pull request merged**